client/my-sites/theme/main.jsx

```diff
@@ -498,17 +498,17 @@ const ThemeSheet = React.createClass( {
 
 		return (
 			<Main className="theme__sheet">
-				<QueryCanonicalTheme themeId={ this.props.id } siteId={ siteID } />
+				<QueryCanonicalTheme themeId={ this.props.id } siteId={ siteId } />
 				{ currentUserId && <QueryUserPurchases userId={ currentUserId } /> }
-				{ siteID && <QuerySitePurchases siteId={ siteID } /> }
-				{ siteID && <QuerySitePlans siteId={ siteID } /> }
+				{ siteId && <QuerySitePurchases siteId={ siteId } /> }
```

Comment:
In a separate PR we should update QuerySitePurchases to do a no-op when given falsey siteId values. Most query component do this since actual usage in the component looks awkward. cc @Tug